dockers/docker-orchagent/supervisord.conf

```diff
@@ -70,6 +70,15 @@ stderr_logfile=syslog
 dependent_startup=true
 dependent_startup_wait_for=orchagent:running
 
+[program:gearsyncd]
+command=/usr/bin/gearsyncd -p /usr/share/sonic/hwsku/gearbox_config.json
```

Comment:
is there a separate PR for gearsyncd?

Reply:
@sujinmkang  Yes, https://github.com/Azure/sonic-swss/pull/1321

Please refer to commit message at top of this page for full list of pull requests. 

This pull request is last, all others are now merged and in fact a part of 202006 release. We did sonic-buildimage last since others are basically dependent.  This one will be backported to the 202006 branch as soon as one of you do a merge.